api/src/pcapi/routes/backoffice_v3/templates/components/forms/postal_address_autocomplete.html

```diff
@@ -0,0 +1,45 @@
+<div class="form-floating input-group{%  if field.hasManualEditing == True %} mt-3{% else %} my-3{% endif %}">
```

Comment:
Tu peux alléger en enlevant les ` == True` et en utilisant `not` plutôt que `== False`, si ce sont des booléens

Reply:
Done